src/lib.rs

```diff
@@ -81,6 +89,8 @@ type Result<T> = std::result::Result<T, DeepgramError>;
 impl Deepgram {
     /// Construct a new Deepgram client.
     ///
+    /// The client will be pointed at the deepgram API.
```

Comment:
Perhaps this should read:
```suggestion
    /// The client will be pointed at the hosted deepgram API.
```

Reply:
I opted for "Deepgram's hosted API".  I think there's slightly more opportunity for confusion between "hosted Deepgram API" and "self-hosted Deepgram API".  (It's a Deepgram API, but who's doing the hosting?) "Deepgram's hosted API" directly answers the question of who's doing the hosting.